config/managers.ini.template

```diff
@@ -2,6 +2,8 @@
 [manifest]
 ;domain = @MANIFEST_SERVICE_HOSTNAME@
 ;domainPort = 80
+[manifest][rendition]
```

Comment:
This is not how node-ini wants it
https://github.com/pastorbones/node-ini

Was considering switching to ini lib , yet will do so at a more ease up time



From: Johnathan Amit Kanarek [mailto:notifications@github.com]
Sent: Wednesday, April 13, 2016 10:48 AM
To: kaltura/play-server <play-server@noreply.github.com>
Cc: Elad Cohen <elad.cohen@kaltura.com>
Subject: Re: [kaltura/play-server] Improve behavior when Wowza restarts (#242)


In config/managers.ini.template<https://github.com/kaltura/play-server/pull/242#discussion_r59505849>:

> @@ -2,6 +2,8 @@

>  [manifest]

>  ;domain = @MANIFEST_SERVICE_HOSTNAME@

>  ;domainPort = 80

> +[manifest][rendition]

This is not how sub-sections should be written in ini file.
Use: rendition.maxRetries = 6 instead.

—
You are receiving this because you modified the open/close state.
Reply to this email directly or view it on GitHub<https://github.com/kaltura/play-server/pull/242/files/3839293a57a33dea4b16ba25dfdd5fc3dd54c81f#r59505849>